### PR TITLE
Responsive members grid and member holo card

### PR DESCRIPTION
**Pull request by brandonfancher**
This is a down payment / progress toward making member list grids responsive and styling the individual member pages. It introduces the `MemberHoloCard` component too...which is sort of a visual representation of the NFT that we'll be able to use elsewhere in the application.

Even though work is not complete on the individual member pages, I wanted to go ahead and PR this so other devs have the option to base on it if desired.

image
image
image
image

### CODE REVIEW

**Comment by brandonfancher** [comment]
Images are a bit heavy. Will address that in subsequent PR.

File: packages/webapp/src/pages/members/[id].tsx [hunk 1]
```diff
@@ -1,27 +1,40 @@
...
-            <MemberCard member={member} />
+        <RawLayout>
+            <Card>
+                <div className="flex justify-center items-center space-y-10 xl:space-y-0 xl:space-x-10 flex-col xl:flex-row">
+                    <div className="max-w-xl">
+                        <MemberHoloCard member={member} />
+                    </div>
+                    <MemberCard member={member} />
+                </div>
+            </Card>
```
Comment (sparkplug0025) [inline packages/webapp/src/pages/members/[id].tsx:26]: What if this entire <Card> is the <MemberCard>? I'm trying to think fo a case where we wnat the <MemberCard> without the HoloCard

File: packages/webapp/src/members/components/members-grid.tsx [hunk 2]
```diff
@@ -1,17 +1,22 @@
 import React from "react";
 import Link from "next/link";
+import dayjs from "dayjs";
+import * as localizedFormat from "dayjs/plugin/localizedFormat";
 
 import { atomicAssets, ipfsBaseUrl } from "config";
 import { MemberData } from "../interfaces";
 import { assetToString } from "_app";
 
+dayjs.extend(localizedFormat.default);
```
Comment (sparkplug0025) [inline packages/webapp/src/members/components/members-grid.tsx:10]: is this dayjs supposed to exist in every file that we reference it or should we add this globally in our config.ts?

File: packages/webapp/src/members/components/members-grid.tsx [hunk 3]
```diff
@@ -21,78 +26,124 @@ export const MembersGrid = ({ members }: Props) => {
...
+            <MemberImage member={member} />
+            <div className="p-3 pt-1">
+                <div className="flex items-center space-x-3 mt-1">
+                    <MintDate createdAt={member.createdAt} />
+                    <AssetBadge member={member} />
+                </div>
+                <MemberNames member={member} />
+                <AuctionBadge member={member} />
+                <SaleBadge member={member} />
+            </div>
```
Comment (sparkplug0025) [inline packages/webapp/src/members/components/members-grid.tsx:43]: beautiful refactor 💯

**Review by sparkplug0025 — APPROVED** [review]
approved! super beautiful improvements! 🥺

File: packages/webapp/src/pages/members/[id].tsx (hunk 1 above)
Comment (brandonfancher) [reply packages/webapp/src/pages/members/[id].tsx:26]: I’m going to be doing some more work on this page today. I’ll def take a look at consolidating this. Thanks!

File: packages/webapp/src/members/components/members-grid.tsx (hunk 2 above)
Comment (brandonfancher) [reply packages/webapp/src/members/components/members-grid.tsx:10]: I think you’re right; prob at the top level. I’ll play with it today.

File: packages/webapp/src/members/components/members-grid.tsx (hunk 3 above)
Comment (brandonfancher) [reply packages/webapp/src/members/components/members-grid.tsx:43]: Thanks!

**Pull request merged**